core/load-git.c

```diff
@@ -722,6 +722,15 @@ static void parse_dc_time(char *line, struct membuffer *str, void *_dc)
 static void parse_dc_watertemp(char *line, struct membuffer *str, void *_dc)
 { (void) str; struct divecomputer *dc = _dc; dc->watertemp = get_temperature(line); }
 
+
+int get_divemode(const char *divemodestring) {
+	for (int i=0; i<6; i++) {
```

Comment:
perhaps we should use a define for `6` everywhere - e.g. `#define DIVEMOD_STRING_TOTAL 6`. please change the name as see fit.